addons/voice/org.openhab.voice.marytts/src/main/java/org/openhab/voice/marytts/internal/MaryTTSAudioStream.java

```diff
@@ -89,7 +89,7 @@ private InputStream getWavHeaderInputStream(int length) throws IOException {
         byte format = 0x10; // PCM
         byte bits = 16;
         byte channel = 1;
-        int srate = 48000;
+        long srate = (this.audioFormat != null) ? this.audioFormat.getFrequency() : 16000l;
```

Comment:
@marcelloceschia  @kaikreuzer for de_DE it is NOT running on my OH2. It's too fast.